src/nupic/bindings/math.i

```diff
@@ -24,6 +24,15 @@
 %include <nupic/bindings/exception.i>
 
 %pythoncode %{
+try:
+	# NOTE need to import capnp first to activate the magic necessary for
```

Comment:
You ended up with tab characters here and elsewhere in this file. Please replace with spaces